### ISSUE TITLE
[BUG] when args.mode == "prompt"

### ISSUE THREAD

**Issue by Himanshunitrr** [jwmao1/MedSegFactory#3]
Not well familiar with the desired flow but there's a bug when args.mode=="prompt".

Current implementation of generate:
```
def generate(
            self,
            keys=None,
            prompts=None,
            negative_prompt=None,
            height=256,
            width=256,
            num_samples=1,
            seed=None,
            guidance_scale=7.5,
            num_inference_steps=50,
            **kwargs,
    ):

```

as we can see prompts is a positional argument therefore this gives error:
`image, label = pipeline.generate(prompt)`

proposed changes:
```
prompt = {'organ':args.organ, 'kind':args.kind, "key":args.key}
image, label = pipeline.generate(prompts=prompt)
```

also even in args.mode=="prompt: you need keys because of :
```
else:
            organ, kind = prompts['organ'], prompts['kind']
            img_prompt = [f'a photo of {organ} image, with {kind}.'] * num_samples
            mask_prompt = [f'a photo of {organ} label, with {kind}.'] * num_samples
            keys = prompts['key']
```

**Comment by jwmao1** [comment]
Thank you for your advice, you are right. Due to our negligence, we are missing the `key` provision. We have now corrected the issue accordingly.

**Comment by Himanshunitrr** [comment]
Thanks, I think you forgot: 
image, label = pipeline.generate(prompts=prompt)

**Comment by jwmao1** [comment]
Yes, we have revised it now.

**Comment by Himanshunitrr** [comment]
Thanks, is there any tentative timeline for the training code :)

**Comment by jwmao1** [comment]
Maybe when the paper is accepted.
